packages/mobile/src/sentry/Sentry.ts

```diff
@@ -33,16 +33,19 @@ export function* initializeSentry() {
     networkConfig.inHouseLiquidityURL,
   ]
 
+  // Disable sentry on e2e tests https://github.com/getsentry/sentry-react-native/issues/1921
   Sentry.init({
     dsn: SENTRY_CLIENT_URL,
     environment: DeviceInfo.getBundleId(),
     enableAutoSessionTracking: true,
-    integrations: [
-      new Sentry.ReactNativeTracing({
-        routingInstrumentation: sentryRoutingInstrumentation,
-        tracingOrigins,
-      }),
-    ],
+    integrations: isE2EEnv
```

Comment:
IMO it would be more straightforward (and understandable to other contributors) to have a "sentryEnabled" flag that's `false` by default and explicitly enabled when we actually want to configure logging to sentry.  That approach makes it safe for anyone to run the repo in any mode w/o worrying about logging to sentry; and the variable rename communicates the semantics (enable/disable sentry) without requiring people to dive into code.

Reply:
@silasbw would we want to have this enabled by default and disabled in e2e tests. I could see an issue where the remoteConfigValues are set to false, and only updated after the remote config fetch leading to missing metrics of app load in Sentry.

Reply:
there is already a `SENTRY_ENABLED` variable that we use to stop Sentry from being initialised completely for dev builds, should we just include the `isE2EEnv` variable there too? what we have here is Sentry minus the performance tracing for e2e tests - have we been using Sentry to evaluate errors generated by E2E tests? to me it's more clear to exclude Sentry completely....

Reply:
Opted to disable sentry entirely during e2e tests. This should reduce noise from sentry to user facing issues.